### PR TITLE
Woudini updates part 2 and more

### PR DESCRIPTION
**Pull request by sragia**
Pictures

chrome_2019-01-08_01-56-09
electron_2019-01-08_01-57-09
electron_2019-01-08_01-57-11
electron_2019-01-08_01-57-15

### CODE REVIEW

**Comment by Stanzilla** [comment]
Lovely! Not quite convinced by the new font yet, could you take a side by side screenshot with old/new fonts? Also which font is the word "companion" in the header?

**Comment by mrbuds** [comment]
```
        Unable to resolve path to module 'v-tooltip'
    
    
         src\renderer\components\LandingPage.vue:119:22
         117 | import path from "path";
         118 | import moment from "moment";
       > 119 | import VTooltip from "v-tooltip";
```

**Comment by mrbuds** [comment]
I really like the new font, old one have some problems

- this "i" has no dot i
- maybe toast was using a different idk but i dislike it toast

**Comment by sragia** [comment]
I posted font comparison in discord day or two ago.  . Can make some more screens if you want. I feel like font much better fits overall theme that is going on her with loads of rounded shapes. Also feel like it's way less pixelated than Noto which for me at least makes whole app look more polished.
Tooltip error - Probably because I didn't commit package.json in as I was changing version there for update testing. 
Can fix this after I am off work today.

**Comment by sragia** [comment]
Also one more thing everything uses same font in app (Or at least it should be). So header logo used Noto Sans before Montserrat now

**Comment by Nightwarden24** [comment]
1. Why are symbols of new font so stretched/expanded? Is it just me that thinks so? Look at the block of text [here](https://user-images.githubusercontent.com/20356414/50800405-bfbc1d80-12e8-11e9-88a5-5ac8be4bb4cb.png)
2. I think you should choose one style for notifications: a dark background & a color icon or a color background & a white icon. The 1st options is better. Besides, the info notification doesn't have a close button. P.s. that blue color is terrible and disgusting.
3. Can anyone finally remove the Discord and Patreon icons from footer? I don't understand why duplicate links if they are already on the About tab.
4. Perhaps you should add more information about the application on the About tab, not just "media" links. It's still a desktop application and not a web one. For example: https://i.imgur.com/o8LtpVK.png

**Comment by Stanzilla** [comment]
> I really like the new font, old one have some problems
> 
> * this "i" has no dot i
> * maybe toast was using a different idk but i dislike it toast

That is because we used Noto *SC*, which means small caps and that the font has no lowercase letters, should look correct with normal Noto.

When I look at the comparison, I like the new one better I think

**Comment by Stanzilla** [comment]
> 2. I think you should choose one style for notifications: a dark background & a color icon or a color background & a white icon. The 1st option is better. Besides, the info notification doesn't have a close button. P.s. that blue color is terrible and disgusting.

Two things here, first yes, I like the bar itself to be the same color at all times and an icon on the left side, what is a problem that you can see on @sragia's last screenshot is that the background color of the toast is the same as the background of the app, that's a visibility problem imo. So, toast background color needs to be a bit brighter than background, "error" style toast needs changed from red to that color and the icon on the error toast needs to be red instead of white.

> 3. Can anyone finally remove the Discord and Patreon icons from footer? I don't understand why duplicate links if they are already on the About tab.

I like the icons there but only because of one thing, symmetry. If we can find something else to put there, I'm all for it!

> 4. Perhaps you should add more information about the application on the About tab, not just "media" links. It's still a desktop application and not a web one. For example: [i.imgur.com/o8LtpVK.png](https://i.imgur.com/o8LtpVK.png)

Yes that looks nice!

**Comment by sragia** [comment]
> 1. Why are symbols of new font so stretched/expanded? Is it just me that thinks so? Look at the block of text [here](https://user-images.githubusercontent.com/20356414/50800405-bfbc1d80-12e8-11e9-88a5-5ac8be4bb4cb.png)
That's just font. But I agree on text blocks it's not the best, but for rest (small phrases) imo it works quite well. Maybe could use different (simillar) font here that looks better
> 2. I think you should choose one style for notifications: a dark background & a color icon or a color background & a white icon. The 1st option is better. Besides, the info notification doesn't have a close button. P.s. that blue color is terrible and disgusting.
"terrible and disgusting" *gasp*. Yeah I think same style could work better here. Info button doesnt have close as it's not really meant to be closed (closes after you have done what it says). However you can swipe it away anyway so maybe actual button would make sense. But yea also what Stan said about changing color a bit to not blend as much with bg.
> 3. Can anyone finally remove the Discord and Patreon icons from footer? I don't understand why duplicate links if they are already on the About tab.
Imo like stan said, something would need to go in their place. But main idea in the start was to have some more important links more prominent in app to direct users there.
> 4. Perhaps you should add more information about the application on the About tab, not just "media" links. It's still a desktop application and not a web one. For example: https://i.imgur.com/o8LtpVK.png
About tab I think has been just a low priority for everyone and no one really has put much thought about what should go in there. Yeah like what you did there, much better than what we have now

File: src/renderer/components/LandingPage.vue [hunk 1]
```diff
@@ -1271,36 +1194,158 @@ a {
...
+    &.default {
+      // Default Toast
+      background-color: rgba(17, 17, 17, 0.95);
+      color: #e6e6e6;
+      font-weight: 500;
+      &:before {
+        content: "";
+        width: 22px;
+        height: 22px;
+        background: url("~@/assets/wow-logo.svg");
```
Comment (mrbuds) [inline src/renderer/components/LandingPage.vue]: wow-logo.svg case doesn't mach with the file so project doesn't compile on mac

**Pull request merged**